catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/data/ContentItem.java

```diff
@@ -63,14 +63,15 @@
     String getUri();
 
     /**
-     * An optional field to provide additional information about a {@link ContentItem}
+     * Return the optional field to describe the {@link ContentItem} as an additional product for a {@link Metacard}.
+     * {@link ContentItem}s added to a {@link Metacard} as an additional product must have a non-blank qualifier.
      *
-     * @return the qualifier of the {@link ContentItem}
+     * @return the qualifier of the {@link ContentItem}. A {@code null} or empty qualifier designates this as the main {@link ContentItem} of the associated {@link Metacard}.
```

Comment:
It seems like qualifier should be an empty Optional when the ContentItem is constructed with a blank qualifier. I don't know how to force every accessor of getQualifier to use the isBlank method instead of .equals("") or .isEmpty(). Is it worth writing up another ticket to address this?